target/step159-2020-1/index.html

```diff
@@ -1,12 +1,92 @@
 <!DOCTYPE html>
```

Comment:
It looks like you have multiple html and js files that are pretty much the same?

Reply:
Yeah, I am not sure how they even got created. I will delete them now and hope they don't autogenerate or something hahaha